Source/WebCore/rendering/style/RenderStyle.h

```diff
@@ -426,7 +426,7 @@ class RenderStyle {
     inline bool hasStaticInlinePosition(bool horizontal) const;
     inline bool hasStaticBlockPosition(bool horizontal) const;
 
-    PositionType position() const { return static_cast<PositionType>(m_nonInheritedFlags.position); }
+    PositionType position() const;
```

Comment:
This should be marked `inline` so we get an error if we forget to include `RenderStyleInlines.h`.